<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fuseHdrBrowser</class>
 <widget class="QMainWindow" name="fuseHdrBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>fuse HDR browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>256</height>
         </size>
        </property>
        <property name="title">
         <string>HDR Preview</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>10</number>
         </property>
         <property name="spacing">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="preview">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>512</width>
             <height>256</height>
            </size>
           </property>
           <property name="text">
            <string>Preview Image</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="vrayEnvCheckbox">
      <property name="text">
       <string>Use in Vray Envirnment instead of a Domelight</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="imageList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="+cc" stdset="0">
         <string>&quot;fuseHdrBrowser.UpdatePreviewImage()&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CreateButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
        <property name="+c" stdset="0">
         <string>&quot;fuseHdrBrowser.CreateDomeLight()&quot;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
